_layouts/contacts.md

```diff
@@ -30,6 +30,26 @@ layout: default
            </a>
          </p>
        {% endif -%}
+        <div id="contact-form">
+          <form action="https://api.web3forms.com/submit" method="POST">
+            <input type="hidden" name="subject" id="form-subject">
+            <input type="hidden" name="access_key" value="{{ site.data.conf.main.contact_form.key }}">
+            <input type="hidden" name="redirect" value="https://curacultura.com">
```

Comment:
Could this be independent from the domain. If the domain changed we would have to change this or it would break.

Reply:
Done with js